app/auth/user_management.py

```
@@ -0,0 +1,79 @@
+    """Update current user's profile information"""
+    try:
+        user_record = auth.get_user(current_user["id"])
+        if update_data.email != user_record.email:
+            auth.update_user(user_record.uid, email=update_data.email)
+        auth.set_custom_user_claims(user_record.uid, {
+            "first_name": update_data.first_name,
+            "last_name": update_data.last_name,
+            "role": current_user.get("role", "user")
+        })
+        return {"message": "Profile updated successfully"}
+    except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)